File: backendAccesos/accesos/models/surtidor.py
from django.contrib.auth.models import User
from django.db import models


class Surtidor(models.Model):
    nombre = models.CharField(max_length=255)
    latitud = models.CharField(max_length=255)
    longitud = models.CharField(max_length=255)
    # n a n con user
    users = models.ManyToManyField(
        User,
        related_name='surtidores_users',
        blank=True,
    )

    def __str__(self):
        return self.nombre

File: frontendConsultas/src/models/Surtidor.ts
export interface Surtidor {
    id: number;
    nombre: string;
    latitud: string;
    longitud: string;
    combustibles: Combustible[];
  }
  
  export interface Combustible {
    id: number;
    tipo: string;
    saldo: number;
    surtidor: number;
    img: string;
  }
